bigquery_etl/glean_usage/templates/clients_last_seen_joined.metadata.yaml

```diff
@@ -16,3 +16,4 @@ bigquery:
   clustering:
     fields:
       - sample_id
+      - normalized_channel
```

Comment:
Stable tables are clustered on `normalized_channel, sample_id` rather than the other way around. I think we should be consistent with that.